docs/06_how-to-guides/40_multi-index/how-to-instantiate-a-multi-index-table.md

```diff
@@ -38,58 +71,68 @@ using namespace eosio;
+    // this data member stores a name for each row of the multi-index table
     name test_primary;
     // additional data stored in table row
     uint64_t datum;
     // mandatory definition for primary key getter
     uint64_t primary_key( ) const { return test_primary.value; }
   };
   
-+  typedef eosio::multi_index<"testtaba"_n, test_table> test_tables;
++  typedef eosio::multi_index<"testtaba"_n, test_table> test_table_t;
 ```
 
-6. Define the multi index table data member of type `test_tables` defined in the previous step
+### 5. Instantiate The Multi-Index Table
+
+Declare the `testtab` multi-index table as a data member of type `test_table_t`.
+
 ```diff
   // the data structure which defines each row of the table
   struct [[eosio::table]] test_table {
-    // this field stores a name for each row of the multi index table
+    // this data member stores a name for each row of the multi-index table
     name test_primary;
     // additional data stored in table row
     uint64_t datum;
     // mandatory definition for primary key getter
     uint64_t primary_key( ) const { return test_primary.value; }
   };
   
-  typedef eosio::multi_index<"testtaba"_n, test_table> test_tables;
-+  test_tables testtab;
+  typedef eosio::multi_index<"testtaba"_n, test_table> test_table_t;
++  test_table_t testtab;
 ```
 
-7. Instantiate the data member `testtab` by passing to its constructor the `scope` (in this case `receiver`) and the `code` parameters, these two combined with table name `"testtaba"` provide access to the partition of the RAM cache used by this multi index table, in this example you will initialize the `testtab` data member in the smart contract constructor
+# 6. Initialize The Multi-Index Table Instance
+
+Initialize the data member `testtab` by passing to its constructor the `scope` (in this case `receiver`) and the `code` parameters, these two combined with table name `"testtaba"` provide access to the partition of the RAM cache used by this multi-index table, in this example you will initialize the `testtab` data member in the smart contract constructor
 
 ```diff
 // contract class constructor
 multi_index_example( name receiver, name code, datastream<const char*> ds ) :
    // contract base class contructor
    contract(receiver, code, ds),
-   // instantiate multi index instance as data member (find it defined below)
+   // instantiate multi-index instance as data member (find it defined below)
 +   testtab(receiver, receiver.value)
    { }
 ```
-Now you have instantiated the `testtab` variable as a multi index table which has a primary index defined for its `test_primary` data member.
 
-Here is how the definition of a `multi_index_example` contract containing a multi index table could look like after following all the steps above.
+Now you have instantiated the `testtab` variable as a multi-index table which has a primary index defined for its `test_primary` data member.
```

Comment:
maybe =>
Now you have instantiated a multi-index table `testtab` variable, `testtab` has a primary index defined on the `test_primary` data member.